src/index.tsx

```diff
@@ -40,14 +40,25 @@ const router = createBrowserRouter(
     )
 )
 
-const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement)
-root.render(
-    <React.StrictMode>
-        <AlertsProvider>
-            <CssReset />
-            <CssVariables colors theme layers spacers elevations />
-            <RouterProvider router={router} />
-            <Alerts />
-        </AlertsProvider>
-    </React.StrictMode>
+function render(tree) {
```

Comment:
What's the purpose of these changes? And are they related to the original issue?

Reply:
Yes, I was logged out immediately because during dev mode, the strict mode is active and executes every hook twice. This caused some weird issues with hooks that are supposed to run once only.

This effects development mode only and is backwardscompatible, so you'd have to actively set the flag for anything to change. But the strictmode can be quite annoying at times

Reply:
Thanks for explaining :pray: